src/vmspawn/vmspawn.c

```diff
@@ -301,9 +312,9 @@ static int parse_argv(int argc, char *argv[]) {
                 }
 
         if (argc > optind) {
-                strv_free(arg_parameters);
-                arg_parameters = strv_copy(argv + optind);
-                if (!arg_parameters)
+                strv_free(arg_kernel_cmdline_extra);
```

Comment:
Why do we have to free the strv here? It should be `NULL` at this point no?

Reply:
I have wondered this too, nspawn did it so I thought it safer to include as well.